frontend/src/app/App.js

```diff
@@ -12,40 +12,52 @@ import { CreateMatchPage } from './pages/CreateMatch'
 import { Profile } from './pages/Profile'
 import { AddPlayerPage } from './pages/AddPlayer'
 import { MatchListPage } from './pages/MatchList'
-import { Button } from '../styles/blocks'
+import { Button, Subtitle } from '../styles/blocks'
+import { isGameSelected } from './modules/games/games-selectors'
+import { connect } from 'react-redux'
 
-export class App extends Component {
+export class AppComponent extends Component {
   render() {
-    const { match: { url } } = this.props
+    const { match: { url, params: { gameName } }, isGameSelected, gameNotFound } = this.props
     const createMatch = () => {
       this.props.history.push(`${url}${ROUTES.CREATE_MATCH}`)
```

Comment:
i know it's not directly related to this PR, but this is kind of an antipattern - because you never know if there's exactly one backslash in url, or routes.creatematch. usually a url is constructed by joining two paths by some third party library that takes care of this, something like url-join etc.
Not a big issue though

Reply:
Add url-join in https://github.com/salsita/foosball-rating/pull/141/commits/65b89d580928f1960523b96e1fa2c2e72be643b5

Reply:
this actually breaks the addMatch redirect completely because of the bug that is solved here https://github.com/salsita/foosball-rating/pull/150